rosidl_generator_py/resource/_msg.py.em

```diff
@@ -246,9 +246,8 @@ if isinstance(type_, NestedType):
         return True
 
     @@classmethod
-    def get_fields_and_field_types(cls):
-        from copy import copy
-        return copy(cls._fields_and_field_types)
+    def get_slot_types(cls):
```

Comment:
This function should probably instead be implemented in the new runtime package proposed in #34.

Reply:
I'll wait for #34 to settle then.
Is there any other changes planned after #34 that would impact this ?

Reply:
Hard to foresee the future :wink: I think this should be able to land right after #34 (even if other stuff comes up).